Review edits to Device Management for Mbed OS (#1046)
ARMmbed/Handbook: docs/pelion_ready/device-management.md

<h1 id="device-management">Device Management for Mbed OS</h1>

> Pelion Client is an abstraction of Device Management Client, and lets Mbed OS devices use the [Pelion Device Management services](https://www.pelion.com/docs/device-management/current/welcome/index.html). Pelion Client:

Reviewer: ```suggestion
Pelion Client is an abstraction of Device Management Client, and lets Mbed OS devices use [Pelion Device Management services](https://www.pelion.com/docs/device-management/current/welcome/index.html). Pelion Client:
```

> - Provides LwM2M Resources - variables that sync automatically through Device Management. Pelion Client makes it trivial to expose sensors, actuators and other variables to a cloud service.

Reviewer: ```suggestion
- Provides LwM2M Resources, which are variables that sync automatically through Device Management. Pelion Client makes it possible to expose sensors, actuators and other variables to a cloud service.
```
Expose?

- [Validation and testing](../mbed-os-pelion/device-management-test.html).
- [Troubleshooting](../mbed-os-pelion/device-management-test.html#troubleshooting).

<span class="tips">**Tip**: To learn more about Device Management Client, see the [API documentation](https://www.pelion.com/docs/device-management/current/client-api-references/index.html) and the [tutorials](https://www.pelion.com/docs/device-management/current/connecting/device-management-client-tutorials.html).</span>

## Device Management for your Mbed OS application

### Requirements

> Not every device (microcontroller, module or board) is capable of running device management features. Although you can add or extend some hardware capabilities, such as connectivity, storage and TRNG. Others are impossible or inconvenient to extend (for example, RAM or flash).

Reviewer: Device management or Device Management? Are we talking general concepts or our product in particular?

> Not every device (microcontroller, module or board) is capable of running device management features. Although you can add or extend some hardware capabilities, such as connectivity, storage and TRNG. Others are impossible or inconvenient to extend (for example, RAM or flash).

Reviewer: ```suggestion
Not every device (microcontroller, module or board) is capable of running device management features. You can add or extend some hardware capabilities, such as connectivity, storage and TRNG. Others are impossible or inconvenient to extend (for example, RAM or flash).
```

The minimum requirements to add device management feature to your application are:

> - RAM: 128 K or more.

Reviewer: KB, no?

> - Flash: 512 K or more.

Reviewer: Same as above.

- Real Time Clock (RTC).

> - (Optional but recommended) True Random Number Generator (TRNG).

Reviewer: ```suggestion
-  True Random Number Generator (TRNG). Optional, but recommended.
```

Reviewer 2: Why rearrange this one?

Author: Either is ok for me.

- A storage device: SD card, SPI flash, QSPI flash or data flash.
- IP connectivity: Ethernet, Wi-Fi, cellular, 6LoWPAN or Thread.

> Additionally, we recommend that the latest version of Mbed OS supports the device and any additional complementary hardware components, or that they have support for the APIs provided in the latest releases of Mbed OS.

Reviewer 2: This "that" is unnecessary.
Query: How formal are we when it comes to the subjunctive mood? Do we keep it as "support," or do we relax it?

Useful references:

- [Supported Mbed OS platforms](https://cloud.mbed.com/quick-start).

> - [Storage options available](../reference/storage.html).

Reviewer 2: Could we get rid of "available" here and just have "Storage options" and "Network options"?

Reviewer: Yes.

Author: Did that already.

Reviewer: Ah, whoops. Sorry!
